ui/admin/tests/acceptance/session-recordings/session-recording/channels-by-connection/channel-test.js

```diff
@@ -146,5 +146,39 @@ module(
       assert.notEqual(currentURL(), incorrectUrl);
       assert.strictEqual(currentURL(), urls.channelRecording);
     });
+
+    test('user cannot view manage dropdown without proper authorization', async function (assert) {
+      // Visit channel
+      featuresService.enable('ssh-session-recording');
+
+      instances.sessionRecording.authorized_actions =
+        instances.sessionRecording.authorized_actions.filter(
+          (item) => item !== 'reapply-storage-policy',
+        );
+      await visit(urls.sessionRecording);
+
+      assert.dom('.hds-menu-primitive').doesNotExist();
+    });
+
+    test('user can view manage dropdown with proper authorization', async function (assert) {
+      // Visit channel
+      featuresService.enable('ssh-session-recording');
+
+      await visit(urls.sessionRecording);
+
+      assert.dom('.hds-menu-primitive').exists();
```

Comment:
Could we update this assert to use a `data-test-*` instead of `.hds-menu-primitive`? That doesn't tell me exactly what we are expecting to see. We can also use `.isVisible()` if this is an element we should be able to see in the window.

Reply:
ty, added